engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -251,7 +251,7 @@ private List<Definition> translateModuleSymbolImpl(Tree inputAst, List<Definitio
           methodRef,
           args,
           body,
-          getIdentifiedLocation(inputAst, 0, 1, null),
+          getIdentifiedLocation(inputAst, 0, 0, null),
```

Comment:
@JaroslavTulach was there any reason why we had this `+1` offset here?

It meant that a one-line method definition like
```
Foo.from (that : Bar) = 42
```
had its span taken _together_ with the newline at the end of the line. This was then causing problems when rendering the error / constructing its associated `SourceSection` with asserts on (violating `SourceSection.isValid()` if that statement was at the end of the source code, because the additional character was getting out of bounds of the source).

It was inconsistent with other entities, like e.g. imports - where I checked that the entity's source section is created _without_ the trailing newline.

----

So I have removed it here. It seems that all tests are passing so I assume it's OK.

I will add a multiline test just to be sure.

Reply:
> span taken _together_ with the newline at the end of the line.

It was necessary to be compatible with the old parser and its tests. Now, when the old parser was removed, we see no inconsistency.